internal/ent/privacy/rule/payment.go

```diff
@@ -0,0 +1,78 @@
+package rule
+
+import (
+	"context"
+	"errors"
+	"slices"
+	"strings"
+
+	"entgo.io/ent"
+
+	"github.com/rs/zerolog/log"
+	"github.com/theopenlane/iam/auth"
+
+	"github.com/theopenlane/core/internal/ent/generated"
+	"github.com/theopenlane/core/internal/ent/generated/organizationsetting"
+	"github.com/theopenlane/core/internal/ent/generated/privacy"
+	"github.com/theopenlane/core/internal/ent/generated/user"
+	"github.com/theopenlane/core/internal/ent/privacy/utils"
+)
+
+var (
+	errNoPaymentMethodAttached = errors.New("a valid payment method is required to create tokens. Contact your organization admin to add one in billing")
```

Comment:
Since this is returned to the user, we want a capital letter on the sentence here:
```suggestion
	errNoPaymentMethodAttached = errors.New("A valid payment method is required to create tokens. Contact your organization admin to add one in billing")
```